fetch/api/cors/cors-preflight-star.any.js

```diff
@@ -35,8 +35,11 @@ function preflightTest(succeeds, withCredentials, allowMethod, allowHeader, useM
 preflightTest(true, false, "get", "x-test", "GET", ["X-Test", "1"])
 preflightTest(true, false, "*", "x-test", "SUPER", ["X-Test", "1"])
 preflightTest(true, false, "*", "*", "OK", ["X-Test", "1"])
+preflightTest(true,  true, "OK", "Content-Type", "GET", ["Content-Type", "uni/corn"])
 preflightTest(false, true, "*", "*", "OK", ["X-Test", "1"])
 preflightTest(false, true, "*", "", "PUT", undefined)
+preflightTest(true,  true, "", "*", "get", undefined)
```

Comment:
Per step 6.5 of https://fetch.spec.whatwg.org/#cors-preflight-fetch-0 and maybe also step 6.3 (not entirely clear how an empty header is supposed to be parsed) this would result in a network error. That it passes in existing user agents is because they don't implement "*" as special syntax.